ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -41,11 +42,35 @@ struct ToDoListView: View {
                         }
                     }
                     .listStyle(.plain)
+                    .onTapGesture {
```

Comment:
I don't think this is expected UX, the button should always be visible for the users, at the same time it shouldn't cover the content of the list when it is scrolled to the bottom. In order to fix that there could be 2 options: apply bottom padding to the list with the same height as the add button. But this can clip the content of the list at the bottom. If that's the case, then another fix would be too add EmptyView to the list after the Todo cells with height equal to the add button frame